neonwranglerpy/lib/retrieve_aop_data.py

```diff
@@ -40,6 +40,8 @@ def retrieve_aop_data(data, year=2019, dpID=['DP3.30006.001'], savepath=""):
         if which_easting.any:
             tiles.loc[which_cheq, "siteID"] = 'CHEQ'
             tiles.drop_duplicates(inplace=True)
+    if isinstance(dpID, str):
+        dpID = [dpID]
```

Comment:
@Nageshbansal could there be a chance that `dpID` is of `NoneType`?

Reply:
`dpID` is not supposed to be `NoneType`, as it takes a default value if no value is passed as argument.

Reply:
Okey, that sounds good if we are sure we won't have that.

I was visioning something like
`data, year, dpiD = Calling_some_function()`
But function fails and returns `(some_data, may_be_a_year, None_for dpiD) `

Using that information to run `retrieve_aop_data` could result into parsing None to the function for dpID.

`retrieve_aop_data(data, year, dpID)`

Reply:
Otherwise, looks good to me

Reply:
Yup. as there is no edge case right now when this case can fail. I'm doing this because I'm iterating over the `dpID` in the [line 47](https://github.com/weecology/neonwranglerpy/blob/948fcc8bbc7f73f7f37db35c8455202a8c0695da/neonwranglerpy/lib/retrieve_aop_data.py#L47).